patterns/base/button/button.twig

```diff
@@ -1,3 +1,3 @@
-{% for button in buttons %}
-  <button class="{{buttons.class}}" type="{{ button.type }}"> {{ button.label }}</button>
+{% for button in versions %}
```

Comment:
no need for a repeat here. The versions are so we can make different version later if we like.